cfgov/v1/models/base.py

```diff
@@ -153,7 +153,7 @@ def related_posts(self, block, hostname):
         for parent_slug, search_type, search_type_name, search_query in search_types:
             try:
                 parent = Page.objects.get(slug=parent_slug)
-                search_query &= Page.objects.descendant_of_q(parent)
+                search_query &= Page.objects.child_of_q(parent)
                 if 'specific_categories' in block.value:
```

Comment:
no, it would return archived pages which is generally something we want to avoid unless we specifically want them